src/Documentation/doc/en/help/design-principles.md

```diff
@@ -1,11 +1,15 @@
-Promote:
+# Design Principles
```

Comment:
I think I would put this at the top level, named in all caps similar to CONTRIBUTING.md. DESIGN_GUIDE send like a reasonable choice. 

The reason is that the stuff in this directory is for user-facing docs, that we show in-game. The directory that you are removing was for developer-facing docs. It was confusingly named, and most of them were obsolete, but it does leave us short of a good place to put such things.